openapi/components/schemas/StorefrontRichBillingPortal.yaml

```
@@ -0,0 +1,57 @@
+type: object
+properties:
+  id:
+    type: string
+    description: ID of the billing portal.
+    readOnly: true
+    maxLength: 50
+    example: bill_prt_0YV7K5TYV5D1P9ZNKDT39KZC3C
+  websiteId:
+    $ref: ./WebsiteId.yaml
+  name:
+    type: string
+    description: Name of the billing portal.
+  status:
+    description: Status of the billing portal.
+    type: string
+    enum:
+      - active
+      - inactive
+  token:
+    description: Session's token used for authentication.
```

nitpick: [possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
    description: Session's token used for authentication.
```